### ISSUE TITLE
Adaptation for another language

### ISSUE THREAD

**Issue by averkij** [booknlp/booknlp#10]
Hello!

Can you advise or get some instructions on how to make this library work with other language, for example Russian?

**Comment by lucy3** [comment]
Yes, [Multilingual BookNLP: Building a Literary NLP Pipeline Across Languages](https://securegrants.neh.gov/publicquery/main.aspx?f=1&gn=HAA-271654-20)

**Comment by averkij** [comment]
This is something about grant, but where is the results?

**Comment by lucy3** [comment]
My best guess is that the grant just suggests that there's maybe work in progress on it

**Comment by averkij** [comment]
Maybe
